src/sandbox/interpolation/atlas-conservative-interpolation.cc

```diff
@@ -205,7 +205,7 @@ int AtlasParallelInterpolation::execute(const AtlasTool::Args& args) {\
             }
         }
         if (type == "CellColumns") {
-            if (!mesh.cells().has_field("lonlat")) {
+            if (! mesh.cells().has_field("lonlat")) {
                 mesh::actions::Build2DCellCentres{"lonlat"}(mesh);
             }
             return functionspace::CellColumns(mesh, option::halo(halo));
```

Comment:
In fact we can first create the CellColumns functionspace which will grow the halo, and then call the mesh action to create the lonlat afterwards.